bats/fb-test-katello.bats

```diff
@@ -4,6 +4,7 @@
 set -o pipefail
 
 @test "check hammer ping" {
+  sleep 60
```

Comment:
It's slightly different. It's an increasing backoff since the sleep time is increased every time. So the sleep times are 1, 2, 3, 4, 5. The 12 comes down to roughly 1 minute.